### PR TITLE
Role Management API v2 (beta)

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Role Management API v2 (beta)

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-485

### CODE REVIEW

File: docs/api/index.md [hunk 1]
```diff
@@ -190,6 +190,11 @@ To connect with other Sumo Logic users, post feedback, or ask a question, visit
   <a href="/docs/api/role-management"><img src={useBaseUrl('img/icons/operations/role-based-access.png')} alt="Thumbnail icon" width="50"/><h4>Roles</h4></a>  
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/api/role-management"><img src={useBaseUrl('img/icons/operations/role-based-access.png')} alt="Thumbnail icon" width="50"/><h4>Roles v2 (Beta)</h4></a>
```
Comment (atishya-22) [inline docs/api/index.md:195]: Should the href field be different for Roles v2 API?

Comment (kimsauce) [inline docs/api/index.md:195]:
```suggestion
  <a href="/docs/api/role-management-v2"><img src={useBaseUrl('img/icons/operations/role-based-access.png')} alt="Thumbnail icon" width="50"/><h4>Roles v2 (Beta)</h4></a>  
```

Comment (kimsauce) [reply docs/api/index.md]: good catch!!

**Review by atishya-22 — APPROVED** [review]
LGTM

**Pull request merged**